README.md

```diff
@@ -1,3 +1,25 @@
 # Project Pythia Portal
 
 ![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/NCAR/pythia-portal/deploy-site/main?logo=github&style=for-the-badge)
+
+This is the source repository for the [Project Pythia portal](https://projectpythia.github.io).
+The portal site is built with [sphinx](https://www.sphinx-doc.org/).
+
+To build the site locally (e.g. for testing new content),
+use [conda](https://docs.conda.io/) to set up a build environment with all dependencies.
+
+First, make a local clone of this source repository on your machine.
+Then, from the `content` directory of the source repository, do this:
+```
+conda env create -f ci/environment.yml
```

Comment:
Should this be...

```suggestion
conda env create -f ../ci/environment.yml
```

Reply:
You are correct on both counts! Thanks for this fix.